sentry/src/main/java/io/sentry/ExternalOptions.java

```diff
@@ -84,8 +84,16 @@ public final class ExternalOptions {
     for (final String inAppExclude : propertiesProvider.getList("in-app-excludes")) {
       options.addInAppExclude(inAppExclude);
     }
-    for (final String tracingOrigin : propertiesProvider.getList("tracing-origins")) {
-      options.addTracingOrigin(tracingOrigin);
+
+    List<String> tracePropagationTargets = propertiesProvider.getList("trace-propagation-targets");
```

Comment:
How do we distinguish between someone trying to specify an empty list (i.e. send nowhere) and the property not being there (i.e. use default and send everywhere). Same question goes for Manifest. Can we check if the property / manifest entry is there and then set an empty list to avoid the default of sending everywhere?